### ISSUE TITLE
Add a "robots together" level

### ISSUE THREAD

**Issue by adamdbrw** [RobotecAI/ROSCon2023Demo#67]
Create a level with following characteristics:
- Similar floor to main level, walls like in main level (but will not be clearly visible), spotlight on smaller area in the middle. Camera looking at the spotlight. In the spotlight, robots arranged in a "V" formation.
- Imported robots. Prioritize robots of exhibitors, especially more recent or more popular models, robots that look good and do not have technical hurdles in importing.
- Make robots look like they real counterparts (also, improve texture quality, ask our dedicated team).
- Make sure to include .repo for source robot descriptions.
- For ROS 1 robots, it might be good to create a docker with Ubuntu 20 and ROS Noetic.
- We need a nice screenshot with 5-7 robots soon.

Follow up work:
- If any work was required after the import, please document the steps and custom work including beautifing. We are building "what next?" and "troubleshooting" documentation for Robot Importer.

**Comment by lewandowskikacper** [comment]
for robotis_op3 (provided in ROS1 package) the following worked:

1. prepare ROS1 docker: `docker pull ros:noetic-robot`
2. `sudo docker run -it ros:noetic-robot`
3.  `sudo apt update && sudo apt install git`
4.  `git clone https://github.com/ROBOTIS-GIT/ROBOTIS-OP3-Common.git`
5.  `cd ROBOTIS-OP3-Common` 
6. `mkdir src && mv * src`    - there is no `src` directory in this repository 
7.  `catkin_make && catkin_make install`
8.  `source install/setup.bash`
9.  `cd src/op3_description/urdf`
10. `xacro robotis_op3.urdf.xacro > out.urdf`
11. copy the `out.urdf` and replace  `robotis_op3.urdf.xacro` with it in afresh repository (https://github.com/ROBOTIS-GIT/ROBOTIS-OP3-Common.git) on your ROS2 machine
12. launch Editor and import the `out.urdf` file with robotImporter

**Comment by jhanca-robotecai** [comment]
Importing log for `ROS 2` robots (ongoing work)
**TLDR**: it works as expected with three main issues
1. Incorrect colors due to lack of texture file (can be copied manually)
2. Incorrect colors due to incorrect material changes
3. Missing inertials causing _libsdformat_ to stop

![Screenshot from 2023-09-06 17-43-47](https://github.com/RobotecAI/ROSCon2023Demo/assets/134940295/411ea67a-8fb6-44ab-9a2f-d63c7f29fc56)


# Index
- [Index](#index)
- [Robots](#robots)
  - [Universal Robots](#universal-robots)
  - [Clearpath Jackal](#clearpath-jackal)
  - [Clearpath Husky](#clearpath-husky)
  - [MangDang Mini Pupper](#mangdang-mini-pupper)
  - [Husarion Rosbot](#husarion-rosbot)

# Robots
Tested on:
- `o3de` _stabilization/2310_ `0be3e09ad6`
- `o3de-extras` _development_ `b5ce30ec`

## Universal Robots

Prepare ROS 2 workspace:

```
mkdir -p testing_ws/src && cd testing_ws/src && \
git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Description.git && \
cd Universal_Robots_ROS2_Description && git checkout 43cfe68f46955e0f71cda3b27504f4e620d1bf5b && \
cd ../.. && colcon build && source install/setup.sh
```

Unmark _Preserve URDF fixed joints_ option.

Set _xacro_ parameters: 
- `name` to `FooRobot` 
- `type` to `ur5e` (possible values: `ur3`, `ur3e`, `ur5`, `ur5e`, `ur10`, `ur10e`, `ur16e`, `ur20`)

Note:
- There are some problems with incorrectly loaded meshes (some are missing). `ur5` looks complete.
- The texture of `ur20` is stored in a separate file, which has to be copied manually at the current stage, [see Github issue](https://github.com/o3de/o3de-extras/issues/477).

## Clearpath Jackal

Prepare ROS 2 workspace:

```
mkdir -p testing_ws/src && cd testing_ws/src && \
git clone https://github.com/jackal/jackal.git && \
cd jackal && git checkout 017b8b581a90873047f7d6fe438bd87513be4a76  && \
cd ../.. && colcon build && source install/setup.sh
```

Note:
- The colors of the model are incorrect, [see Github issue](https://github.com/o3de/o3de-extras/issues/489).

Unmark _Preserve URDF fixed joints_ option.

## Clearpath Husky

Prepare ROS 2 workspace:

```
mkdir -p testing_ws/src && cd testing_ws/src && \
git clone https://github.com/husky/husky.git && \
cd husky && git checkout 1e0b1d14d657f04ec3a86e73d6676a2cf7af6f79  && \
cd ../.. && colcon build && source install/setup.sh
```

Unmark _Preserve URDF fixed joints_ option.

## MangDang Mini Pupper

Prepare ROS 2 workspace:

```
mkdir -p testing_ws/src && cd testing_ws/src && \
git clone https://github.com/mangdangroboticsclub/mini_pupper_ros.git && \
cd mini_pupper_ros && git checkout 276875c09f3423cf307a2b0d0320944e41612712 && \
cd ../.. && colcon build && source install/setup.sh
```

Unmark _Preserve URDF fixed joints_ option.

Note:
- The colors of the model are incorrect, [see Github issue](https://github.com/o3de/o3de-extras/issues/489).

## Husarion Rosbot

Prepare ROS 2 workspace:
```
mkdir -p testing_ws/src && cd testing_ws/src && \
git clone https://github.com/husarion/rosbot_xl_ros.git && cd rosbot_xl_ros && git checkout 94795ff30585b2b6faa595cd6519a8f9ca17f5d0 && cd .. && \
git clone https://github.com/husarion/ros_components_description.git && cd ros_components_description && git checkout 5f94c332c866c8e39f6854f77d98651a2a3ae226 && cd .. && \
cd .. && colcon build && source install/setup.sh
```

Unmark _Preserve URDF fixed joints_ option.
